apps/ui/assets/.env.example

```diff
@@ -0,0 +1,2 @@
+BASE_URL=http://localhost:4000
+GIPHY_API_KEY=
```

Comment:
@dlederle Hit me up if you need this value to fill in when you `cp .env.example .env`.

Reply:
yes please!